examples/tv-casting-app/darwin/MatterTvCastingBridge/MatterTvCastingBridge/CastingServerBridge.mm

```diff
@@ -495,6 +510,79 @@ - (void)shutdownAllSubscriptions:(dispatch_queue_t _Nonnull)clientQueue requestS
     });
 }
 
+- (void)initMatterServer
```

Comment:
since this is re-connecting to a video player that it was previously connected to, it implies this is intended to be called more than once

I suggest we rename this to startMatterServer or something instead even if we're calling the init function internally

Its kind of weird to call the init function more than once

Reply:
Sure, I'll change it to `startMatterServer`

Reply:
See https://github.com/project-chip/connectedhomeip/pull/24046/commits/1b8dcc16ea1800e40dd7b0d2a82548dfdf731bb1